addons/account/models/account_report.py

```diff
@@ -600,10 +600,10 @@ def _compute_auditable(self):
     @api.constrains('engine', 'report_line_id')
     def _validate_engine(self):
         for expression in self:
-            if expression.engine == 'aggregation' and (expression.report_line_id.groupby or expression.report_line_id.user_groupby):
+            if expression.engine in ('aggregation', 'external') and (expression.report_line_id.groupby or expression.report_line_id.user_groupby):
                 raise ValidationError(_(
-                    "Groupby feature isn't supported by aggregation engine. Please remove the groupby value on '%s'",
-                    expression.report_line_id.display_name,
+                    "Groupby feature isn't supported by %s engine. Please remove the groupby value on '%s'",
+                    expression.engine, expression.report_line_id.display_name,
```

Comment:
Use the translated user string in user facing messages
```python
engine_description = dict(expression._fields['engine']._description_selection(self.env))
```

```suggestion
                    engine_description[expression.engine],
                    expression.report_line_id.display_name,
```